time stamp
float64 px
float64 py
float64 vx
float64 vy
float64 ux_nom
float64 uy_nom
float64 ux_safe
float64 uy_safe